data/campaigns/Liberty/scenarios/04_Unlawful_Orders.cfg

```diff
@@ -606,6 +617,205 @@
         [/message]
     [/event]
 
+    #
+    # These two events creates enemy naga units that were pillaging the ship
```

Comment:
naga units?

Reply:
I initially thought to put naga there, and it looks like I forgot to change it.